packages/adapter-sveltekit/src/hooks/documentation-read.ts

```diff
@@ -69,6 +65,10 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 						page,
 					};
 				}
+
+				export async function entries() {
+					return [{}]
```

Comment:
@angeloashmore This looks like it will always return the root, `/`? Should it be something like the repeatable one?
```
export async function entries() {
	const client = createClient();

	const page = await client.getSingle("${model.id}");

        return { uid: page.uid };
}
```

Reply:
From what I can tell, `{}` doesn't mean root. Instead, it means there are no route parameters to define for the current route.

In this case, because the document is a singleton, it likely does not contain a UID and the route will likely not use a dynamic parameter. Thus, `{}` is used for a route like `src/routes/[[preview=preview]]/foo/bar/+page.server.js`.

Like the repeatable page snippet, we don't include `preview: "preview"` because we want it to be server rendered.

Reply:
I tested it locally and it works as it should: `/foo/bar` is prerendered and `/preview/foo/bar` is server-rendered.

Reply:
@angeloashmore Ah, I understand! Sounds good.